test/Microsoft.DotNet.Tools.Tests.Utilities/RepoDirectoriesProvider.cs

```diff
@@ -55,20 +55,17 @@ static RepoDirectoriesProvider()
                 }
                 else
                 {
-                    string configuration = new DirectoryInfo(AppContext.BaseDirectory).Parent.Name;
+                    // https://stackoverflow.com/a/60545278/294804
+                    var assemblyConfigurationAttribute = typeof(RepoDirectoriesProvider).Assembly.GetCustomAttribute<AssemblyConfigurationAttribute>();
+                    string configuration = assemblyConfigurationAttribute?.Configuration;
```

Comment:
Simplification as the parent directory's name is not the same between the Installer repo and SDK repo. The point of this is to simply get the build configuration. So, I directly get that information from the assembly information.